src/main/kotlin/pt/up/fe/ni/website/backend/dto/account/UpdateAccountDto.kt

```diff
@@ -0,0 +1,15 @@
+package pt.up.fe.ni.website.backend.dto.account
+
+import java.util.Date
+import pt.up.fe.ni.website.backend.dto.entity.CustomWebsiteDto
+
+class UpdateAccountDto(
```

Comment:
It might be confusing to have AccountDto when it's only used for creating it. I suggest changing it to CreateAccountDto (similarly to update)